pkgs/top-level/all-packages.nix

```diff
@@ -6281,6 +6281,8 @@ in
 
   cgdb = callPackage ../development/tools/misc/cgdb { };
 
+  cheat = callPackage ../data/documentation/cheat { };
```

Comment:
I moved it to `applications/misc` now, it does definitely not belong to `application/office`. Its core asset are actually these short text files containing documentation like `manpages` or `gnome-user-docs`.